src/metabase/actions.clj

```diff
@@ -147,12 +144,15 @@
   [action-or-id]
   (check-actions-enabled-for-database! (api/check-404 (database-for-action action-or-id))))
 
-(defn perform-action!
+(mu/defn perform-action!
   "Perform an `action`. Invoke this function for performing actions, e.g. in API endpoints;
   implement [[perform-action!*]] to add support for a new driver/action combo. The shape of `arg-map` depends on the
   `action` being performed. [[action-arg-map-spec]] returns the specific spec used to validate `arg-map` for a given
   `action`."
-  [action arg-map]
+  [action
```

Comment:
As chronicled [here](https://metaboat.slack.com/archives/C04DN5VRQM6/p1710449728575199), the Actions stuff used to accept either keywords or strings for column names. That made normalization hard to test, since it needs to decide between one or the other. Keyword column names are icky anyway, since column names can be blank, or have spaces, or slashes, so I updated all the Actions code to use string names and only string names.